balancer/rls/config.go

```diff
@@ -148,6 +157,14 @@ func (rlsBB) ParseConfig(c json.RawMessage) (serviceconfig.LoadBalancingConfig,
 		return nil, err
 	}
 
+	if sc := string(cfgJSON.RouteLookupChannelServiceConfig); sc != "" {
+		parsed := internal.ParseServiceConfig.(func(string) *serviceconfig.ParseResult)(sc)
+		if parsed.Err != nil {
+			return nil, fmt.Errorf("rls: bad control channel service config %s: %v", sc, parsed.Err)
```

Comment:
`%q` might be a little nicer than `%s` here.

Reply:
Done.